packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -299,74 +306,49 @@ type DestinationCertificateJson = {
 
 async function callCertificateEndpoint(
   uri: string,
-  headers: Record<string, any>,
-  options?: ResilienceOptions
+  headers: Record<string, any>
 ): Promise<AxiosResponse<DestinationCertificateJson>> {
   if (!uri.includes('Certificates')) {
     throw new Error(
       `callCertificateEndpoint was called with illegal arrgument: ${uri}. URL must be certificate endpoint of destination service.`
     );
   }
-  return callDestinationService(uri, headers, options) as Promise<
+  return callDestinationService(uri, headers) as Promise<
     AxiosResponse<DestinationCertificateJson>
   >;
 }
 
 async function callDestinationEndpoint(
   uri: string,
-  headers: Record<string, any>,
-  options?: ResilienceOptions
+  headers: Record<string, any>
 ): Promise<AxiosResponse<DestinationJson | DestinationConfiguration>> {
   if (!uri.match(/[instance|subaccount]Destinations|v1\/destinations/)) {
     throw new Error(
       `callDestinationEndpoint was called with illegal arrgument: ${uri}. URL must be destination(s) endpoint of destination service.`
     );
   }
-  return callDestinationService(uri, headers, options) as Promise<
+  return callDestinationService(uri, headers) as Promise<
     AxiosResponse<DestinationConfiguration | DestinationJson>
   >;
 }
 
 async function callDestinationService(
   uri: string,
-  headers: Record<string, any>,
-  options?: ResilienceOptions
+  headers: Record<string, any>
 ): Promise<
   AxiosResponse<
     DestinationCertificateJson | DestinationConfiguration | DestinationJson
   >
 > {
-  const { enableCircuitBreaker, timeout } = {
-    ...defaultResilienceBTPServices,
-    ...options
-  };
   const config: AxiosRequestConfig = {
     ...urlAndAgent(uri),
     proxy: false,
     method: 'get',
-    timeout,
+    timeout: 10000, // TODO: Use middleware
```

Comment:
todos will be addressed in another pr? maybe reference the issue where it will be done here?